Data/models/BraveTodaySourceMO.swift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import CoreData
+
+public final class BraveTodaySourceMO: NSManagedObject, CRUD {
+    @NSManaged public var enabled: Bool
+    @NSManaged public var publisherID: String
+    
+    public class func get(fromId id: String) -> BraveTodaySourceMO? {
+        getInternal(fromId: id)
+    }
+    
+    public class func all() -> [BraveTodaySourceMO] {
+        all() ?? []
+    }
+    
+    public class func setEnabled(forId id: String, enabled: Bool) {
+        setEnabledInternal(forId: id, enabled: enabled)
+    }
+    
+    public class func setEnabled(forIds ids: [String], enabled: Bool) {
+        DataController.perform(context: .new(inMemory: false)) { context in
+            ids.forEach {
+                setEnabledInternal(forId: $0, enabled: enabled, context: .existing(context))
+            }
+        }
+    }
+    
+    public class func resetSourceSelection() {
```

Comment:
This should also be called from 'Clear private data' menu

Reply:
I asked if source selection should be cleared from Clear Private Data section and they said no